satellite/payments/coupons.go

```diff
@@ -19,7 +19,7 @@ type Coupons interface {
 	ListByUserID(ctx context.Context, userID uuid.UUID) ([]Coupon, error)
 
 	// Create attaches a coupon for payment account.
-	Create(ctx context.Context, coupon Coupon) (err error)
+	Create(ctx context.Context, coupon Coupon) (coup Coupon, err error)
```

Comment:
"coup" could be named as "created"

Reply:
I'll change that in the PR that adds the remove functionality.